src/services/billing/cost-management/cost-dashboard/CostDashboardPage.vue

```diff
@@ -158,17 +159,30 @@ export default {
             SpaceRouter.router.push({ name: BILLING_ROUTE.COST_MANAGEMENT.DASHBOARD.CUSTOMIZE._NAME, params: { dashboardId: props.dashboardId } });
         };
 
-        const fetchLayoutData = async (layoutId: string): Promise<any[]> => {
+        const fetchDefaultLayoutData = async (layoutId: string): Promise<any[]> => {
             try {
                 // noinspection TypeScriptCheckImport
-                const res = await import(`./dashboard-layouts/${layoutId}.json`);
-                return res.default;
+                const layoutTemplates = await import(`./dashboard-layouts/${layoutId}.json`);
+                const widgets = await import('../widgets/lib/defaultWidgetList.json');
+
+                const optionsKeyByWidgetId = keyBy(widgets.default, option => option.widget_id);
+                const layoutData: CustomLayout[] = layoutTemplates.default.map(layout => layout.map(d => ({ ...optionsKeyByWidgetId[d.widget_id] })));
```

Comment:
```suggestion
                const layoutData: CustomLayout[] = layoutTemplates.default.map(layout => layout.map(d => {
                const widget = optionsKeyByWidgetId[d.widget_id]
                return widget ? { ...widget } : {}
                }));
```

`optionsKeyByWidgetId[d.widget_id]`가 `undefined`인 경우에 대한 가딩이 필요합니다.